templates/accounts/detail.html

```diff
@@ -1,7 +1,10 @@
 {% extends 'base.html' %}
 {% block title %}{{ user.username }} detail{% endblock %}
 {% block content %}
-<h1>{{ user.username }}</h1>
-<p>Email: {{ user.email }}</p>
-<p>Name: {{ user.first_name }} {{ user.last_name }}</p>
+    <h1>{{ user.username }}</h1>
+    <p>Email: {{ user.email }}</p>
```

Comment:
nie wiedzę tłumaczeń w templatkach, sprawdź czy wszystkie dodałeś, i to pewnie można przez bootstrapa fajnie wyświetlić przez tabelkę